### PR TITLE
Add metrics e2e test

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Adding an e2e test case about metrics retrieval.
#### Which issue(s) this PR fixes:

Refers to #276 
#### Does this PR have test?
Yes

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
Should be rebased on top of https://github.com/kubernetes-sigs/security-profiles-operator/pull/429

**Comment by saschagrunert** [comment]
This is ready for review. PTAL @cmurphy @JAORMX

**Comment by saschagrunert** [comment]
@JAORMX @cmurphy need another lgtm here

File: test/e2e_test.go [hunk 1]
```diff
@@ -78,6 +78,10 @@ func (e *e2e) TestSecurityProfilesOperator() {
 			"Seccomp: Delete profiles",
 			e.testCaseDeleteProfiles,
 		},
+		{
+			"Seccomp: Metrics",
+			e.testCaseMetrics,
+		},
```
Comment (JAORMX) [inline test/e2e_test.go:84]: oho! I just noticed, the re-deploy test needs to go on spot 4 in the array. See line 128.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: test/e2e_test.go (hunk 1 above)
Comment (saschagrunert) [reply test/e2e_test.go:84]: Oops, I changed it to `5`.

**Pull request merged**